app.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Financial-Times/go-fthealth/v1a"
+	"github.com/Financial-Times/http-handlers-go/httphandlers"
+	status "github.com/Financial-Times/service-status-go/httphandlers"
+	log "github.com/Sirupsen/logrus"
+	"github.com/gorilla/mux"
+	"github.com/jawher/mow.cli"
+	"github.com/rcrowley/go-metrics"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+)
+
+func main() {
+	app := cli.App("curated-authors-transformer", "A RESTful API for transforming Bertha Curated Authors to UP People JSON")
+
+	port := app.Int(cli.IntOpt{
+		Name:   "port",
+		Value:  8080,
+		Desc:   "Port to listen on",
+		EnvVar: "PORT",
+	})
+	berthaSrcUrl := app.String(cli.StringOpt{
+		Name:   "bertha-source-url",
+		Value:  "http://bertha.ig.ft.com/view/publish/gss/1wEdVRLtayZ6-XBfYM3vKAGaOV64cNJD3L8MlLM8-uFY/TestAuthors",
```

Comment:
Does Bertha have authentication? Because if not, we are making it easy for people to find this data by publishing the api url. 

Probably OK now, but I can imagine it would be easy to add new fields that we don't want everyone to see... and not realise.